contents/data_vis.md

```diff
@@ -523,12 +541,193 @@ Next, we will dive into how to manage and control **layouts**.
 
 ### Layouts {#sec:makie_layouts}
 
-- Overview on several ways to do layouts and related attributes.
+A complete _canvas/layout_ is defined by `Figure`, which can be fill up afterwards with content. 
+We will start with a simple arrangment of one `Axis`, one `Legend` and one `Colorbar`. 
+For this task we can think of the canvas as an arrangment of `rows` and `columns`. 
+The `Axis` content will be in _row 1, column 1_, e.g. `fig[1,1]`, the `Colorbar` in _row 1, column 2_, namely `fig[1,2]`.
```

Comment:
```suggestion
The `Axis` content will be in _row 1, column 1_, e.g. `fig[1, 1]`, the `Colorbar` in _row 1, column 2_, namely `fig[1,2]`.
```

Reply:
Since this is just regular indexing of an object like in many places in Julia, it's probably more easy readable for people when there is a space behind the comma